src/Kernel.ImageSupport/Helpers/ImageResizeHelper.cs

```diff
@@ -9,8 +9,9 @@
 using SixLabors.ImageSharp.Processing;
 using Svg;
 using Image = SixLabors.ImageSharp.Image;
+using Rectangle = SixLabors.ImageSharp.Rectangle;
 
-namespace LT.DigitalOffice.Kernel.ImageSupport.Helpers;
+namespace DigitalOffice.Kernel.ImageSupport.Helpers;

```

Comment:
better stay with "LT" or we would have to change it in all services, where use it